lib/internal/bootstrap/node.js

```diff
@@ -70,6 +70,7 @@ const rawMethods = internalBinding('process_methods');
 // Set up methods and events on the process object for the main thread
 if (isMainThread) {
   process.abort = rawMethods.abort;
+  process.triggerFatalException = rawMethods.triggerFatalException;
```

Comment:
Can you use `Object.defineProperty` to make this read-only? I am kind of afraid of this being monkey-patchable in the wild and that people count on this being called by core, then it will be another can of worms..